resources/server/match/match.service.ts

```diff
@@ -42,21 +43,29 @@ class _MatchService {
   }
 
   async handleSaveLikes(reqObj: PromiseRequest<Like[]>, resp: PromiseEventResp<boolean>) {
-    const identifier = PlayerService.getIdentifier(reqObj.source);
-    matchLogger.debug(`Saving likes for identifier ${identifier}`);
+    const player = PlayerService.getPlayer(reqObj.source);
+    matchLogger.debug(`Saving likes for identifier ${player.getIdentifier()}`);
 
     try {
-      await this.matchDB.saveLikes(identifier, reqObj.data);
+      await this.matchDB.saveLikes(player.getIdentifier(), reqObj.data);
     } catch (e) {
       matchLogger.error(`Failed to save likes, ${e.message}`);
       resp({ status: 'error', errorMsg: 'GENERIC_DB_ERROR' });
     }
 
     try {
-      const newMatches = await this.matchDB.findNewMatches(identifier, reqObj.data);
+      const newMatches = await this.matchDB.findNewMatches(player.getIdentifier(), reqObj.data);
```

Comment:
Please refactor to call player.getIdentifier() once.

```ts
const playerIdent = player.getIdentifier()
```

Reply:
Fixed in next commit.